src/auro-pane.js

```diff
@@ -24,6 +24,7 @@ import styleCssFixed from './style-fixed-css.js';
  * @attr {Boolean} fixed - uses px values instead of rem for fonts.
  * @attr {String} price - Sets price for display. Displayed as is.
  * @attr {Boolean} selected - Sets pane state to selected.
+ * @attr {Boolean} md - Sticks the component to the mobile breakpoint.
```

Comment:
I am not following the use case? The implemented code is not referencing a breakpoint, but describing a new UI display size?